docs/approvals/service-now/index.md

```diff
@@ -0,0 +1,151 @@
+---
+title: Service Now Integration
+description: Approvals, approvals, approvals
+position: 10
+---
+:::warning
+The ServiceNow Integration feature is only available as an EAP for version 2022.2.5556 or later and it requires a feature license to be able to use it.
+:::
+
+## Overview
+
+The Octopus Deploy/ServiceNow integration allows users to block the execution of specifically configured deployments unless they have a corresponding approved ServiceNow **Change Request** (CR).
+
+For configured project and environment combinations, Octopus Deploy will 
+
+To enable this behavior, both systems must be configured before deployments can be managed.
+
+| Project | Environment Change Controlled | Environment **Not** Change Controlled| 
+| --------|--|-- |
+| Change Controlled | _Approval Required_  | No Approval Required | 
+| **Not** Change Controlled | No Approval Required | No Approval Required |
+
+## Getting started
+
+The Service Now integration requires Octopus 2022.2.5556 or later, and an Octopus license with the Service Now Integration feature enabled.
+
+To get started:
+1. Configure ServiceNow
+1. Obtain a ServiceNow feature license for your Octopus instance 
+1. Configure a connection from Octopus to ServiceNow
+1. Configure which deployments require an approved CR
+
+### Configuring Service Now
+
+:::hint
+The instructions in this section will require a ServiceNow Administrator.
+:::
+
+The Octopus Deploy / ServiceNow integration requires security configuration in your target ServiceNow instance. 
+
+Follow the [ServiceNow OAuth documentation](https://docs.servicenow.com/bundle/sandiego-platform-administration/page/administer/security/task/t_SettingUpOAuth.html) to configure an OAuth endpoint for Octopus to use for authentication. Take note of the OAuth client id and client secret from the configuration.
+
+Next, the integration will require a user account on ServiceNow. The recommendation is to create a user account specifically for Octopus.
+To create a new ServiceNow user, follow the ServiceNow [Create a user](https://docs.servicenow.com/en-US/bundle/sandiego-platform-administration/page/administer/users-and-groups/task/t_CreateAUser.html) documentation. 
+
+Ensure that the new user has `Web service access only` checked. 
+Take note of the password assigned or generated for this user.
+
+### Licensing
+
+For the ServiceNow approval checks to be performed as part of the deployment process, an appropriate Octopus license must be configured in your Octopus instance.
+
+A ServiceNow enabled Octopus license must be requested from Octopus directly, and cannot be managed through the usual self-service process.
```

Comment:
```suggestion
A ServiceNow enabled Octopus license must be requested from Octopus directly, and cannot be managed through the self-service process.
```

Reply:
Should we include the link to register for EAP here? The registration form is the entry point for the customers to ask for a licence. See [SNOW EAP program](https://docs.google.com/document/d/1U1HA9GQxqcU9tAZIkRAUSjQGX0SC1EpbJg7_tos8d84/edit#bookmark=id.y9docr7ru9ke)

Reply:
added link to EAP registration